geniza/corpus/annotation_export.py

```diff
@@ -365,17 +373,18 @@ def annotation_list_name(canvas_uri):
         return "_".join([prefix, path])
 
     @staticmethod
-    def transcription_filename(document, source):
+    def filename(document, source, fn_type):
         # filename based on pgpid and source authors;
-        # explicitly label as transcription for context
+        # explicitly label as transcription/translation for context
         authors = [a.creator.last_name for a in source.authorship_set.all()] or [
             "unknown author"
         ]
 
-        return "PGPID%s_s%d_%s_transcription" % (
+        return "PGPID%s_s%d_%s_%s" % (
             document.id,
             source.id,
             slugify(" ".join(authors)),
+            fn_type,
         )
```

Comment:
This is enough parts that it's getting hard to read, recommend switching to named placeholders `%(id)s` and a dictionary. Maybe something like this? (not tested)

```suggestion
        return "PGPID%(pgpid)s_s%(source_id)d_%(authors)s_%(text_type)s" % {
            "pgpid": document.id,
            "source_id": source.id,
            "authors": slugify(" ".join(authors)),
            "text_type": fn_type,
        }
```